Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -345,7 +345,7 @@ static inline ASAuthorizationPublicKeyCredentialLargeBlobSupportRequirement toAS
     for (auto credential : options.allowCredentials) {
         if (credential.transports.contains(AuthenticatorTransport::Internal) || credential.transports.isEmpty())
             [platformAllowedCredentials addObject:adoptNS([allocASAuthorizationPlatformPublicKeyCredentialDescriptorInstance() initWithCredentialID:toNSData(credential.id).get()]).get()];
-        if (credential.transports.isEmpty() || !credential.transports.contains(AuthenticatorTransport::Internal)) {
+        if (credential.transports.isEmpty() || credential.transports.containsIf([](auto transport) { return transport != AuthenticatorTransport::Internal && transport != AuthenticatorTransport::Hybrid; })) {
```

Comment:
I wonder if this new logic (neither `Internal` not `Hybrid`) could be represented as `isNotPlatformRequest(credential.transports)`?

That might be slightly easier to understand when reading, but it's just a suggestion.